pkg/scalers/gcp_storage_scaler.go

```diff
@@ -193,7 +193,7 @@ func (s *gcsScaler) GetMetricsAndActivity(ctx context.Context, metricName string
 // getItemCount gets the number of items in the bucket, up to maxCount
 func (s *gcsScaler) getItemCount(ctx context.Context, maxCount int64) (int64, error) {
 	query := &storage.Query{Delimiter: s.metadata.blobDelimiter, Prefix: s.metadata.blobPrefix}
-	err := query.SetAttrSelection([]string{"Name"})
+	err := query.SetAttrSelection([]string{"Size"})
```

Comment:
why this?

Reply:
This is because  `SetAttrSelection` as a reduction filter to limit the retrieved information. We are retrieving just the name which isn't used at all, as the real filtering is executed on gcp side, and I need the size of the blob to discard folders (folders have size 0). As `Name` isn't used and I need the `Size`, I just swapped them instead of adding `Size` as extra field